setup/docker.rst

```diff
@@ -0,0 +1,56 @@
+.. index:: Docker
+
+Using Docker with Symfony
+=========================
+
+Can you use Docker with Symfony? Of course! And several tools exist to help,
+depending on your needs.
+
+Complete Docker Environment
+---------------------------
+
+If you'd like a complete Docker environment (i.e. where PHP, web server, database,
+etc. are all in Docker), check out `https://github.com/dunglas/symfony-docker`_.
+
+Alternatively, you can install PHP on your local machine and use the
+:ref:`symfony binary Docker integration <symfony-server-docker>`. In both cases,
+you can take advantage of automatic Docker configuration from Flex.
```

Comment:
Maybe we can do this change to avoid confusion for newcomers:

```rst
[...] from Flex.
```

-->

```rst
[...] from :ref:`Symfony Flex <symfony-flex>`.
```